clients/application-kafka/src/test/java/org/eclipse/hono/application/client/kafka/impl/KafkaApplicationClientImplTest.java

```diff
@@ -136,6 +147,63 @@ public void testCloseConsumer(final Type msgType, final VertxTestContext ctx) {
                 })));
     }
 
+    /**
+     * Verifies that a command sent asynchronously to a device succeeds.
+     *
+     * @param ctx The vert.x test context.
+     */
+    @Test
+    public void testSendAsyncCommandSucceeds(final VertxTestContext ctx) {
```

Comment:
shoudn't these tests better be in *KafkaBasedCommandSenderTest*?

Reply:
I have moved these tests to the `KafkaBasedCommandSenderTest` and also the tests for the proton based implementation to the `ProtonBasedCommandSenderTest`.